### ISSUE TITLE
Definicion de Estados discretos

### ISSUE THREAD

**Issue by SantiagoOsorio3** [IIC1253/IIC1253-2024-1#63]
Queria preguntar que al momento de definir P (el conjunto de todos los estados discretos). Puedo en el mismo conjunto mezclar a dos estados? A lo que me refiero es si es que tengo variables que dependan de dos estados en paralelo como i y j por ejemplo. Puedo definir el conjunto mezclando las variable de dos estados distintos en el mismo conjunto ?

**Comment by AndreaPGS** [comment]
Sí, está bien definir un conjunto de estados y luego otros conjuntos diferentes que mezclen estos estados. De hecho, un ejemplo es como se modela el ejercicio del mapa 3-coloreable.
